lib/teselagen/examples/pytested/CRISPR-Tool.ipynb

```diff
@@ -67,15 +59,42 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Connection Accepted\n"
+      "Connection Accepted\n",
+      "Selected Lab: Test Lab\n"
      ]
     }
    ],
    "source": [
     "# Connect to your teselagen instance by passing it as the 'host_url' argument of TeselaGenClient(host_url=host_url)\n",
+    "# client = TeselaGenClient(host_url=\"https://your-instance-name.teselagen.com\")\n",
     "client = TeselaGenClient()\n",
     "client.login()\n",
-    "client_discover = client.discover"
+    "client_discover = client.discover\n",
+    "client.select_laboratory(lab_name=\"Test Lab\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Display *host_url*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://platform.teselagen.com\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(client.host_url)"
```

Comment:
Just as a comment for future developments:

I would like to suggest that we evaluate developing _Jupyter Notebooks_ in _Visual Studio Code_:

1. because it has the advantages of being able to use robust _linters_ and _debugging tools_, _auto complete_,  and _formatters_, by means of the [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) extension ( which includes both [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) and [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) extensions ) 

1. because we can develop them as python scripts, which both _simplifies_ and _strengthens_ the _code review_ process.

1. because [we can transform from `.py` files to `.ipynb` (and viceversa) and export them automatically](https://code.visualstudio.com/docs/python/jupyter-support-py#_export-a-jupyter-notebook) in _Visual Studio Code_:
    * which can be done as follows:
         1. `cmd + shift + P`
         1. `Jupyter: Restart Kernel`
         1. `Jupyter: Run All Cells`
         1. `Jupyter: Export Current Python File and Output as Jupyter Notebook`
        
        (we could probably create a task or a shortcut to do it in one step)